simply_fit/simply_fit_api/views.py

```diff
@@ -70,8 +70,23 @@ def getUserWorkOutHoursForDate(user, date):
 				return log
 		else:
 			raise KeyError("User does not exist! Please add User first")
-
-
+	'''
+	The purpose of this method is to update a users information.
+	@param  user=string
+		new_firstName=string
+		new_lastName=string
+		
+	@return
+		None
+	'''
+	@staticmethod
+	def updateUserInfo(user, new_firstName=None, new_lastName=None):
```

Comment:
Would this method be another way for user to add a new user into the system?

Thanks